services/provider/server/server.go

```diff
@@ -611,10 +617,14 @@ func (s *OCSProviderServer) GetStorageClaimConfig(ctx context.Context, req *pb.S
 				rbdStorageClassData["encryptionKMSID"] = storageRequest.Spec.EncryptionMethod
 			}
 
+			rbdStorageClassMetaData := map[string]string{
+				"mirroring": fmt.Sprintf("%v", cbp.Spec.Mirroring.Enabled),
+			}
```

Comment:
If I am not wrong this will always be false, as you are haven't fetched the cbp

Reply:
> You are getting the rns here, not the cbl, is this intended?

CBPRN and it's respective CPB carry the same name.